### PR TITLE
Limit linewidth to approx 79 in all python code blocks

### PR DESCRIPTION
**Pull request by hdashnow**
Addresses #225 (except for URLs)

### CODE REVIEW

**Comment by stefanv** [comment]
Maybe we should get my htmlbook_fixes branch merged before doing any further editorial work?  That branch at least compiles to HTMLBook, but it has some finicky parts.  E.g., lines cannot start with `$`!

**Comment by hdashnow** [comment]
Sure, just leave this hanging and I'll rebase once HTMLBook is merged.

**Comment by jni** [comment]
@hdashnow time to rebase! =)

**Comment by hdashnow** [comment]
Rebased

File: markdown/ch8.markdown [hunk 1]
```diff
@@ -731,7 +731,7 @@ def genome(file_pattern):
 Let's try it out on the Drosophila (fruit fly) genome:
 
 ```python
-# dm6.fa.gz can be downloaded from ftp://hgdownload.cse.ucsc.edu/goldenPath/dm6/bigZips/
+# Downloaded dm6.fa.gz from ftp://hgdownload.cse.ucsc.edu/goldenPath/dm6/bigZips/
```
Comment (stefanv) [inline markdown/ch8.markdown]: Downloaded -> Download

**Comment by hdashnow** [comment]
Thanks @stefanv, fixed.

**Comment by stefanv** [comment]
Thanks!

**Pull request merged**